packages/vue/scripts/build-web-types.js

```diff
@@ -0,0 +1,83 @@
+const fs = require("fs")
+
+// Web-types build require docs to be built first
+const docs = require("../../../docs/core.json")
```

Comment:
```suggestion
const docs = require('@ionic/core/dist/docs.json')
```

Let's use the docs.json file output by ionic core as that is what we use in the other copy scripts (same exact file, just loading from a different location).